edge/pkg/devicetwin/dtmanager/membership.go

```diff
@@ -65,7 +66,96 @@ func initMemActionCallBack() {
 	memActionCallBack[dtcommon.MemGet] = dealMembershipGet
 	memActionCallBack[dtcommon.MemUpdated] = dealMembershipUpdate
 	memActionCallBack[dtcommon.MemDetailResult] = dealMembershipDetail
+	memActionCallBack[dtcommon.MemDeviceCRDInsert] = dealDeviceCrdInsert
+	memActionCallBack[dtcommon.MemDeviceCRDDelete] = dealDeviceCrdDelete
+	memActionCallBack[dtcommon.MemDeviceCRDUpdate] = dealDeviceCrdUpdate
 }
+
+func dealDeviceCrdInsert(context *dtcontext.DTContext, resource string, msg interface{}) (interface{}, error) {
+	klog.Infof("Membership event")
```

Comment:
V(4)

Reply:
ok, got it

Reply:
Unify them, now there is not a common rule for log level.